requirements.txt

```diff
@@ -3,3 +3,9 @@ click==8.1.3
 python-dateutil==2.8.2
 tqdm==4.65.0
 colorama==0.4.6
+
+boto3==1.34.80
+botocore==1.34.80
```

Comment:
this is probably redundant, as it's a dependency of boto3

Reply:
Done